README.md

```diff
@@ -1 +1,241 @@
+```
+<!--
+### Install
+
+Run this command: 
+
+Example command:
+
+```sh
+  cd space-travelers-hub
+  npm install
+```
+-->
+
+### Usage
+
+Run this command: 
+- Start Ruby ird
+- Run ruby my_list.rb
+<!--
+Example command:
+
+```sh
+  cd space-travelers-hub
+  npm start
+```
+
+
+### Run tests
+
+Example command:
+
+```sh
+  cd space-travelers-hub
+  npm test
+```
+
+### Deployment
+
+
+You can deploy this project using:
+
+Visual studio code, another realiable code editor or a Web browser
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+ -->
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+<h4>Ritika Rawat</h4>
+
+- GitHub: [ritikarawat220](https://github.com/ritikarawat220)
+- Twitter: [@ritikarawat22](https://twitter.com/Ritikarawat22)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/rawatritika/)
+
+👤 **Eduardo Villarreal**
+
+- GitHub: [@evillaz](https://github.com/evillaz)
+- Twitter: [@evillazz](https://twitter.com/evillazz)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/eduardo-villarreal-144a8925a)
+
+👤 **Tshobohwa Salomon**
+
+- GitHub: [@Tshobohwa](https://github.com/Tshobohwa)
+- Twitter: [@SalomonTshoboh1](https://twitter.com/SalomonTshoboh1)
+- LinkedIn: [@Tshobohwa](https://www.linkedin.com/in/tshobohwasalomon/)
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- **[Not for now]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project  don't doubt on make any comments or try to reach me out.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+<!-- Give credit to everyone who inspired your codebase. -->
+
+I would like to thank microverse for the opportunity on working each other as colleagues and the learning opportunities it brings us.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
+
+## ❓ FAQ (OPTIONAL) <a name="faq"></a>
+
+- **Did you find hard to create this project?**
+
+  - I tried my best and follow the guidelines to create this project. I got hard times on some of the workflow but i think i just have to keep learning everyday and give my best try every time.
+
+- **What was your thinking process to solve the requirements?**
+
+  - First i tried to address each one of the requirements and solve them individually to keep integrating them as i was progressing.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](/LICENSE.md) licensed.
```

Comment:
Please kindly add a LICENSE to your project to fix the broken link 👍

![image](https://github.com/evillaz/enumerable/assets/78251927/7b0444af-7753-4132-8438-af7b86e3efba)

Public repositories on GitHub are often used to share open-source software. For your repository to be open source, you'll need to license it so that others are free to use, change, and distribute the software. 🤗